index.js

```diff
@@ -17,14 +24,11 @@ var _ = require('underscore');
 function Heartbeat(baseUrl, pulse = 60) {
```

Comment:
baseUrl and pulse not used anymore
You probably didn't change it since it is used all over the place - but at least some comment maybe?

Reply:
Yep, exactly. Will add a comment